cloud/store/store.go

```diff
@@ -205,6 +220,35 @@ func (st *Store) Open(ctx context.Context) (err error) {
 		st.loadDatabase(ctx)
 	}
 
+	existedConfig, err := st.configViaPeers(filepath.Join(st.raftDir, peersFileName))
```

Comment:
do you mean store in the `raft.db` ? I assume the `raft.db` file will be alway up to date with Join/Leave

Reply:
Peers.json

Reply:
it doesn't need to because the file will be generated from `raft.db` on start

Reply:
It would be more efficient to store it on graceful shutdown so that we don't have to query the raft.db at all

Reply:
how we would detect the candidates if not gonna either query `raft.db` or keep a list in memory of join/leave nodes and create the file on graceful shutdown ! 

the current impl. considering raft.db, alternative impl. would be keeping the list in memory and create the file on graceful shutdown, the pitfall i see of keeping in memory in case of the node crashed without graceful shutdown that won't be recoverable and would need manual intervention